### ISSUE TITLE
IPA Biweekly Meeting Agenda Request - Rust Code Walk Through

### ISSUE THREAD

**Issue by bmcase** [patcg-individual-drafts/ipa#76]
## Agenda+: What do you want to discuss?

We'll give a walk through of the code for the MPC to help other folks interested in ramping up to the repo. 
 https://github.com/private-attribution/ipa

## Time

30 min

**Comment by bmcase** [comment]
@NigelNTU , our next bi-weekly call will be June 14 and we will likely cover it then.  
> I see that it was supposed to be held in the previous meeting that was cancelled?

Yes, we did cancel the last meeting as quite a few folks couldn't make it and we didn't have an agenda lined up.  We added this item after canceling to discuss on a following call.  We had some discussion on agenda items in our W3C slack channel if you'd like to join there: [w3ccommunity.slack.com](http://w3ccommunity.slack.com/) channel #patcg-ipa
